controller/src/test/java/io/pravega/controller/timeout/TimeoutServiceTest.java

```diff
@@ -507,11 +477,6 @@ private TxnId convert(UUID uuid) {
     }
 
     private <T> void checkError(CompletableFuture<T> future, Class<? extends Throwable> expectedException) {
-        try {
-            future.join();
-            Assert.assertTrue(false);
-        } catch (CompletionException ce) {
-            Assert.assertEquals(expectedException, ce.getCause().getClass());
-        }
+        AssertExtensions.assertThrows("Failed future", future, e -> e.getClass().equals(expectedException));
```

Comment:
No, expectedException is only a variable of type Class, `e instanceOf expectedException` does not even compile.